daemon/test/scripts/helpers.sh

```diff
@@ -188,6 +188,14 @@ start_lightningd()
 
     PORT2=`get_info_field "$LCLI2" port`
     [ $NUM_LIGHTNINGD = 2 ] || PORT3=`get_info_field "$LCLI3" port`
+
+    # Make a payment into a P2SH for anchor.
+    P2SHADDR=`$LCLI1 newaddr | sed -n 's/{ "address" : "\(.*\)" }/\1/p'`
+    FUND_INPUT_TXID=`$CLI sendtoaddress $P2SHADDR 0.01`
+    FUND_INPUT_TX=`$CLI getrawtransaction $FUND_INPUT_TXID`
```

Comment:
Erk, no.  In shell you return things via stdout, and there is other output here.  Maybe one day we'll rewrite in python, though that's probably overkill...